packages/stark-ui/assets/styles/components/_button-theme.scss

```diff
@@ -27,6 +27,10 @@
     .mat-ripple-element {
       background-color: rgba($color: $color, $alpha: 0.2);
     }
+
+    & svg {
+      fill: $color;
```

Comment:
I don't think this is needed, because Angular Material defines this: `fill: currentColor`... so we can simply remove this ;)